packages/components/navbar/src/NavbarAccount/NavbarAccount.styles.ts

```diff
@@ -0,0 +1,44 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+import { NavbarAccountProps } from './NavbarAccount';
+
+export const getNavbarAccountStyles = () => ({
+  root: css({
+    // default button reset styles
+    margin: 0,
+    padding: 0,
+    background: 'none',
+    border: 'none',
+
+    cursor: 'pointer',
+    position: 'relative',
+    alignItems: 'center',
+    justifyContent: 'center',
+    gap: tokens.spacing2Xs,
+
+    '&:focus-visible': {
```

Comment:
We should add the focus fallback we've been using in other components here. At least for now, maybe we can move away from it in the future.